nixos/modules/virtualisation/digital-ocean-config.nix

```diff
@@ -0,0 +1,153 @@
+{ config, pkgs, lib, modulesPath, ... }:
+with lib;
+{
+  imports = [
+    (modulesPath + "/profiles/qemu-guest.nix")
```

Comment:
The only reason I used modules path in my version is because I try to avoid `<Nixpkgs/` as that is not set in my environment (I don't use nix-channel). Not sure if it's worse or better than using relative paths or Nixpkgs paths.

Reply:
I think it is better, because it will work just fine if you are building from a channel, but if you are building NixOS from a custom package set or git checkout, especially via the `nixos` attribute in pkgs/top-level/all-packages.nix, it will do the right thing even without a nixpkgs element in NIX_PATH